cluster/gce/gci/mounter/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:jessie
+MAINTAINER vishh@google.com
+
+RUN apt-get update && apt-get install -y nfs-common glusterfs-client
```

Comment:
Are we sure this `glusterfs-client` is recent enough? I ran into op-version compatibility issues when testing gluster on Ubuntu where the client version was too far behind the server contained in `gcr.io/google_containers/volume-gluster:0.2`. I was able to get around it in my Ubuntu testing by adding a ppa (`sudo add-apt-repository ppa:gluster/glusterfs-3.7`, https://launchpad.net/~gluster/+archive/ubuntu/glusterfs-3.7) and installing the client from there.

I imagine this would depend on whether the APT sources for Jessie have a recent enough version.

To clarify, in my experience the client installed via apt was version 3.4.2 while the server in the above container is version 3.7.4. Version 3.4.2 sets the max op-version to 2, and is incompatible with version 3.7.4 because the op-version ticked to 3+ somewhere in between.